hystrix/settings.go

```diff
@@ -101,3 +101,11 @@ func getSettings(name string) *settings {
 
 	return s
 }
+
+func GetCircuitSettingsMap() map[string]*settings{
```

Comment:
exported functions should only return exported types.  in this case, `Settings` should also be exported.